content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -9,61 +9,71 @@ weight: 20
 
-* Cluster-scoped subresource: `GET /apis/GROUP/VERSION/RESOURCETYPE/NAME/SUBRESOURCE`
-* Namespace-scoped subresource: `GET /apis/GROUP/VERSION/namespaces/NAMESPACE/RESOURCETYPE/NAME/SUBRESOURCE`
+* Cluster-scoped subresource: `GET /apis/<GROUP>/<VERSION>/<RESOURCETYPE>/<NAME>/<SUBRESOURCE>`
+* Namespace-scoped subresource: `GET /apis/<GROUP>/<VERSION>/namespaces/<NAMESPACE>/<RESOURCETYPE>/<NAME>/<SUBRESOURCE>`
 
 The verbs supported for each subresource will differ depending on the object - see the API documentation more information. It is not possible to access sub-resources across multiple resources - generally a new virtual resource type would be used if that becomes necessary.
 
 
 ## Efficient detection of changes
 
-To enable clients to build a model of the current state of a cluster, all Kubernetes object resource types are required to support consistent lists and an incremental change notification feed called a **watch**.  Every Kubernetes object has a `resourceVersion` field representing the version of that resource as stored in the underlying database. When retrieving a collection of resources (either namespace or cluster scoped), the response from the server will contain a `resourceVersion` value that can be used to initiate a watch against the server. The server will return all changes (creates, deletes, and updates) that occur after the supplied `resourceVersion`. This allows a client to fetch the current state and then watch for changes without missing any updates. If the client watch is disconnected they can restart a new watch from the last returned `resourceVersion`, or perform a new collection request and begin again. See [Resource Version Semantics](#resource-versions) for more detail.
+To enable clients to build a model of the current state of a cluster, all Kubernetes object resource types are required to support consistent lists and an incremental change notification feed called a **watch**.  Every Kubernetes object has a `resourceVersion` field representing the version of that resource as stored in the underlying database. When retrieving a collection of resources (either namespace or cluster scoped), the response from the server will contain a `resourceVersion` value that can be used to initiate a Watch against the server. The server will return all changes (creates, deletes, and updates) that occur after the supplied `resourceVersion`. This allows a client to fetch the current state and then watch for changes without missing any updates. If the client Watch is disconnected they can restart a new Watch from the last returned `resourceVersion`, or perform a new collection request and begin again. See [Resource Versions](#resource-versions) for more detail.
 
 For example:
 
 1. List all of the pods in a given namespace.
 
-   ```console
-   GET /api/v1/namespaces/test/pods
-   ---
-   200 OK
+   ```
+   GET /api/v1/namespaces/test/pods HTTP/1.1
+   …more headers…
```

Comment:
Please avoid using "…". Use "...".

Reply:
> Please avoid using … (a single character). Use ... (three dots) when appropriate.

I recommend …; what's the shortcoming there?

Reply:
it is not ASCII character

Reply:
@tengqm I've no problem with Unicode - we already have 13 localizations and several of those rely on UTF-8 being widely adopted. What's the risk with using characters beyond ASCII's 127?

Reply:
This paragraph is a fenced code block, although it is not code strictly speaking.
In a code block, we rarely see "…", we only see plain dots.

Reply:
I've omitted this change from the latest revision.